contracts/stake/src/state.rs

```diff
@@ -27,14 +28,14 @@ use transfer_contract_types::*;
 #[derive(Debug, Default, Clone)]
 pub struct StakeState {
     stakes: BTreeMap<[u8; PublicKey::SIZE], StakeData>,
-    owners: BTreeSet<[u8; PublicKey::SIZE]>,
+    slashed: u64,
```

Comment:
Maybe `slashed_amount` ? Or add a comment above

Reply:
changed to `slashed_amount`